dev/tests/integration/testsuite/Magento/Catalog/Model/ProductRepositoryTest.php

```diff
@@ -233,4 +233,24 @@ public function testCustomLayout(): void
         }
         $this->assertTrue($caughtException);
     }
+
+    /**
+     * Tests product repository update should use provided store code.
+     *
+     * @magentoDataFixture Magento/Catalog/_files/product_simple.php
+     */
+    public function testProductUpdate(): void
+    {
+        $sku = 'simple';
+        $nameUpdated = 'updated';
+
+        $product = $this->productRepository->get($sku, false, 0);
```

Comment:
Replace magic number 0 with it's actual meaning in const - like `STUB_STORE_ID_GLOBAL`

Reply:
done